rcl_action/include/rcl_action/goal_handle.h

```diff
@@ -190,6 +191,56 @@ rcl_action_goal_handle_get_status(
   const rcl_action_goal_handle_t * goal_handle,
   rcl_action_goal_state_t * status);
 
+/// Get the goal terminal timestamp.
+/**
+ * This is a non-blocking call.
+ *
+ * <hr>
+ * Attribute          | Adherence
+ * ------------------ | -------------
+ * Allocates Memory   | No
+ * Thread-Safe        | No
+ * Uses Atomics       | No
+ * Lock-Free          | No
```

Comment:
? I do not see any lock mechanism in there...?

```suggestion
 * Lock-Free          | Yes
```

Reply:
Updated